frontend/src/lib/components/AppEditorLink/appUrlsLogic.ts

```diff
@@ -85,28 +76,29 @@ export const appUrlsLogic = kea({
         suggestions: [
             [],
             {
-                [actions.addUrl]: (state, { value }) => [...state].filter((item) => value !== item),
+                addUrl: (state, { value }) => [...state].filter((item) => value !== item),
             },
         ],
     }),
-
-    listeners: ({ values, sharedListeners, props }) => ({
+    listeners: ({ values, sharedListeners, props, actions }) => ({
         addUrlAndGo: async ({ value }) => {
-            let app_urls = [...values.appUrls, value]
-            await api.update('api/user', { team: { app_urls } })
-            window.location.href = appEditorUrl(props.actionId, value)
+            // TODO: Need to refactor this to use `teamLogic.actions.updateCurrentTeam`
```

Comment:
Figured this was already quite a large PR, but will tackle it already

Reply:
I'm actually thinking it might make sense to leave as is, otherwise we have to add extra logic (as we had before, comment below), to track what was updated in the `updateTeamSuccess` listener.